DTS-STN/Alpha-Site
# Ie not supported

# Description

[DEV Handling IE not being supported](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-67)

This PR adds the `/notsupported` page which a user will be redirected to ONLY when they are using Internet Explorer. This is achieved by using the Redirects function in our next.config. Since most of our styling isn't compatible at all with IE (thanks Tailwind), I've also added a bunch of IE-only CSS to make it look the same. This comes with its quirks (see Help Requested section) but overall it works fine.

## Acceptance Criteria

When a user attempts to navigate to our site using Internet Explorer, they are redirected to a page telling them their browser isn't supported with information on how to download a supported browser.

## Test Instructions

1. Pull in branch
2. Type `npm run dev`
3. Launch Internet Explorer and navigate to `http://localhost:3000/`
4. You should be redirected to `/notsupported`
5. Click on the button that says `Copy link` (give it a second, IE seems to struggle but it does eventually copy the link to your clipboard) and paste it in your address bar to confirm the link was properly copied.
6. Click the links to make sure you're properly redirected to each browsers' download site

## Help Requested

Not that it's a huge issue since this page will only be presented to users on older browsers, but for whatever reason when the window is a certain size, there will be a very small amount of horizontal scroll. I assume there's some padding or margins doing some funky stuff somewhere but I haven't been able to spot where.

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96)
